src/main/java/com/ksm/wstbackoffice/exception/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.ksm.wstbackoffice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
+    @ExceptionHandler(ResourceNotFoundException.class)
+    public ResponseEntity<ErrorResponseMessage> resourceNotFoundException(ResourceNotFoundException ex, WebRequest request) {
+        ErrorResponseMessage message = new ErrorResponseMessage(
+                HttpStatus.NOT_FOUND.value(),
+                new Date(),
```

Comment:
I know we used LocalDate instead of Date.
Can you tell me why it's better to use LocalDate and the main differences between the two?

Reply:
changed to LocalDate

Date class 
	contains both date and time
	not thread safe
	does not support time zones
LocalDate class
	contain just date and doesn't have any time part
	thread safe
	support time zones

Reply:
Why do you think the client wants to see the date when receiving NOT_FOUND?

Reply:
date can be useful for log info in feature